qemu/tests/cfg/hv_flag_cpuid_check.cfg

```diff
@@ -0,0 +1,22 @@
+- hv_flag_cpuid_check:
+    type = hv_flag_cpuid_check
+    only x86_64
+    only Linux
+    required_qemu = [8.0.0,)
+    cpuid_chk_cmd = "rpm -qa | grep cpuid"
+    cpuid_pkg = ""
+    variants:
+        - hv_tlbflush_ext:
+            hv_flag = "hv_tlbflush_ext"
+            cpu_model_flags += ",hv-tlbflush-ext,hv-tlbflush,hv-vpindex"
+            check_cpuid_entry_cmd = "cpuid -r -1 -l 0x40000003"
+            check_register = "edx"
+            check_bit = 14
+            hv_flags_to_ignore = hv-tlbflush-ext
+        - hv_tlbflush_direct:
+            hv_flag = "hv_tlbflush_direct"
+            cpu_model_flags += ",hv-tlbflush-direct,hv-vapic"
```

Comment:
@peixiu  we don't have to add hv-vapic here for it is a default flag already added in avocado.

internal_cfg/qemu-build/qemu-kvm.cfg:13:        cpu_model_flags += ",hv_stimer,hv_synic,hv_vpindex,hv_relaxed,hv_spinlocks=0x1fff,**hv_vapic**,hv_time,hv_frequencies,hv_runtime"

Reply:
Hi @menli820 

Thanks for you propoal, It seems AMD has some issues, none this flag on the AMD host, so I added it here, in order to make sure this case needed hv-flag all are attached. These 2 flags is bounded for this case.

Thanks~
Peixiu

Reply:
As we talked privately， it is reasonable. so no changed need.